util/action.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$id": "https://kulturkrock.se/action.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Action in Opus",
```

Comment:
Is it necessary to mention that it's in an opus (and in Screencrash, below this)? When it's used in the opus schema I think it's clear from the context.